app/jobs/risc_delivery_job.rb

```diff
@@ -0,0 +1,53 @@
+class RiscDeliveryJob < ApplicationJob
+  queue_as :low
+
+  retry_on Faraday::TimeoutError, Faraday::ConnectionFailed, wait: :exponentially_longer
```

Comment:
https://edgeapi.rubyonrails.org/classes/ActiveJob/Exceptions/ClassMethods.html#method-i-retry_on

The defaults here are 5 retries, and with this exponential config will go from about 3s delay to a few minutes

I think this OK for now? open to suggestions